src/routes/api/request.routes.js

```diff
@@ -12,5 +12,7 @@ const {
 const router = express.Router();
 
 router.post('/', [isCurrentUserLoggedIn, isCurrentUserVerified, validateTripRequest], createTripRequest);
+router.get('/most-traveled-destinations', isCurrentUserLoggedIn, isCurrentUserLoggedIn, placesAndVisitTimes);
```

Comment:
1. Could you rename this route to maybe `/trips/statistics`?
2. Why have you put the same middleware `isCurrentUserLoggedIn` twice?

Reply:
1. If I change this route to `/trips/statistics` , what about this task ? 
![Screenshot from 2020-03-23 09-40-54](https://user-images.githubusercontent.com/56886064/77293291-b307c780-6cea-11ea-9aca-b509dcd97b41.png)
I think it the one with statistics, so it would bring some confusion, maybe if you have any other suggestion apart from statistics I would follow that.

2. Adding `isCurrentUserLoggedIn` twice, was a mistake, I change it

Reply:
Ok. You're right. It makes sense to leave it as `/most-traveled-destinations`